R/gh.R

```diff
@@ -71,12 +71,28 @@ maybe <- function(expr, otherwise, quiet = TRUE) {
 
 #' @importFrom git2r remote_url repository discover_repository
```

Comment:
looks like you'll also want `remotes` in the `@importFrom` list and then re-run `devtools::document()` to update the `NAMESPACE`

Reply:
The `git2r::remotes()` call here is qualified so I think we're OK?

Reply:
right, I believe R CMD check won't complain since it's qualified already, but I believe a package should still explicitly declare its imported functions in `NAMESPACE`, just for consistency of having those explicit, but perhaps that's silly.